### DISCUSSION TITLE
Create Variable block could create variable from functions

### DISCUSSION THREAD

**Discussion by mootensai** [postmanlabs/postman-flows#52] [Q&A] [answered]
Before Postman Flow exists, I use Pre-Request Script to set variables like this:

var uuid = require('uuid');
pm.collectionVariables.set("orderId", uuid.v4());
pm.collectionVariables.set("transactionId", uuid.v4());

how to do this in Postman Flow?

**Comment by anudeepmedicharla** [comment, marked as answer]
Hey @mootensai, 
we're planning to introduce a faker like library to perform these actions and generate dynamic values. Will let you know once that feature is out :)

**Reply by itseramin** [reply]
Any updates on this? It has been 4 months, and I really need this feature...
